src/styles/rhd-theme/components/_cards.scss

```diff
@@ -72,15 +75,27 @@
 .rhd-c-card {
   position: relative;
   padding: 0;
+  box-shadow: none;
+  border: 2px solid $light-gray;
   hr {
     border-top: var(--rhd-theme--border-width-md) solid #ededed; // #ededed
-    margin: var(--pf-global--spacer--sm) var(--pf-global--spacer--md) 0; // 8px 16px 0 16px
+    margin: var(--pf-global--spacer--sm) 0; // 8px 16px 0 16px
```

Comment:
Commented sizes are incorrect - should be `// 8px 0`

Reply:
Good spot, will do